dali/operators/image/crop/bbox_crop.cc

```diff
@@ -201,9 +211,29 @@ E.g ``ltrb=True`` is equivalent to ``bbox_layout="xyXY"`` and ``ltrb=False`` cor
         true)
     .AddOptionalArg(
         "num_attempts",
-        R"code(Number of attempts to get a crop window that matches the ``aspect_ratio`` and selected
-IoU value from ``thresholds``.)code",
+        R"code(Number of attempts to get a crop window that matches the ``aspect_ratio`` and a selected
+IoU value from ``thresholds``.
+
+After each ``num_attempts``, a different threshold will be picked, until
+reaching a maximum of ``total_num_attempts``.)code",
         1)
+    .AddOptionalArg(
+        "total_num_attempts",
+        R"code(Total number of attempts to get a crop window that matches the ``aspect_ratio`` and any selected
+IoU value from ``thresholds``.
+
+After each ``num_attempts``, a different threshold will be picked, until
+reaching a maximum of ``total_num_attempts``.
+
+If not specified, ``total_num_attempts`` will be set to
+``num_attempts * num_thresholds * 10``.)code",
```

Comment:
What happens after `total_num_attempts`?

Reply:
One thought from me about this. One thing I see here is that it changes how this op behaves. There might be some code in the wild (MLPerf) that works now, that will fail with this change.
Maybe we should make it as an optional behavior? Or make an option of disabling it at least?
As far as I remember, for SSD there was never a problem with this being indefinite, but I don't remember any measures how many tries it takes to find proper crop. We only did some histograms of what thresholds were chosen, I think. I suspect that how long it takes is very dependent of exact situation.